templates/CRM/Case/Form/AddContact.tpl

```diff
@@ -0,0 +1 @@
+{include file="CRM/Caseroles/Form/AddToCaseAsRole.tpl"}
```

Comment:
This file seems quite useless. What's the purpose of not putting the contents of `templates/CRM/Case/Form/AddToCaseAsRole.tpl` here?